components/list/list-item-mixin.js

```diff
@@ -337,6 +337,13 @@ export const ListItemMixin = superclass => class extends composeMixins(
 			:host([_hovering-selection]) [slot="outside-control-container"] {
 				box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
 			}
+			:host([current="page"]) [slot="outside-control-container"] {
+				margin-block: 1px;
+				outline: 3px solid var(--d2l-button-focus-color, var(--d2l-color-celestine));
+			}
+			:host([current="page"]) [slot="control-container"]::after {
+				border-color: transparent;
+			}
```

Comment:
Not official styles, just wanted an indicator of current similar to in lessons

Reply:
Should they be in the list-item-menu-item-mixin?